README.md

```diff
@@ -6,22 +6,13 @@ A CLI for the Jamsocket platform
 
 # Install
 
-You may run the Jamsocket CLI without explicitly installing it using `npx jamsocket`. However, this may lead to CLI-version-related issues as [NPX is not guaranteed to run the latest version](https://github.com/npm/cli/issues/4108) and will instead use whatever version was cached with the first usage.
-
-For this reason it is recommended that you first install the Jamsocket CLI either globally or in your project and _then_ use `npx jamsocket`. (When running `npx jamsocket` from your project, the locally-installed `jamsocket` will be used.) When installing the Jamsocket CLI globally, it is recommended you use a tool like [Node Version Manager](https://github.com/nvm-sh/nvm#installing-and-updating) to manage your NodeJS binaries. NVM makes sure these binaries are installed under your user so global npm installs do not require root privileges. Once NVM is set up, run `npm install jamsocket --global`.
-
-# Upgrading
-
-```sh
-npm install jamsocket@latest # to update the locally-installed Jamsocket
-npm install jamsocket@latest --global # to update the global installation of Jamsocket
-```
+We recommend you run the Jamsocket CLI without explicitly installing it using `npx jamsocket`. You may also install it locally within a project or globally using `npm install jamsocket` (and `npm install jamsocket -g`).
```

Comment:
can we just omit the "and npx install jamsocket -g" part? If I didn't know what '-g' did I'd be confused whether I need to run both with and without the -g, or why I would use the -g in the first place. We could elaborate, but since -g is discouraged, probably better to omit